ProjFS.Mac/PrjFSKextTests/KauthHandlerTests.mm

```diff
@@ -263,4 +263,25 @@ - (void)testShouldHandleVnodeOpEvent {
     XCTAssertEqual(kauthResult, KAUTH_RESULT_DEFER);
 }
 
+//KEXT_STATIC void UseMainForkIfNamedStream(vnode_t& vnode, bool& putVnodeWhenDone);
+- (void)testUseMainForkIfNamedStream {
+    shared_ptr<mount> testMount = mount::Create();
+    const char* filePath = "/Users/test/code/Repo/file";
+    shared_ptr<vnode> testFileVnode = testMount->CreateVnodeTree(filePath);
+
+    // Out Parameters to test
+    bool result;
```

Comment:
Naming suggestion: `result` made me think this value represents if the call succeeded or failed, when it actually corresponds to the `putVnodeWhenDone` parameter.

Maybe rename `result ` to `putVnodeWhenDone`?